### ISSUE TITLE
Migration problem with

### ISSUE THREAD

**Issue by zardilior** [sdispater/orator#295]
Migration failing in
`table.enum('columna',['0','1','2'])`
to
`table.enum('columna',['0','1','2','3']).change()`

**Comment by rfschubert** [comment]
Yeah currently is a known bug... You will need update your database manually.
If you find how to update Orator and do a PR will be awesome.
